allennlp/modules/stacked_bidirectional_lstm.py

```diff
@@ -0,0 +1,116 @@
+from typing import Optional, Tuple
+import torch
+from torch.nn.utils.rnn import PackedSequence, pack_padded_sequence, pad_packed_sequence
+from allennlp.modules.augmented_lstm import AugmentedLstm
+from allennlp.common.checks import ConfigurationError
+
+
+class StackedBidirectionalLstm(torch.nn.Module):
+    """
+    A standard stacked Bidirectional LSTM where the LSTM layers
+    are concatenated between each layer.
```

Comment:
You should probably note what the differences are between this and the standard LSTM, and why you might want to use one or the other.